frontend/src/components/Review.jsx

```diff
@@ -54,36 +55,88 @@ const VoteDiv = styled.div`
 
 const UpVote = styled(ArrowUpwardIcon)`
   padding-left: 10px;
-  color: #f6da73;
-  transform: scale(1.4);
+  color: ${({ vote }) => (vote === "up" ? "#f6da73" : "#3e5336")};
+  transform: ${({ vote }) => (vote === "up" ? "scale(2.0)" : "scale(1.4)")};
 `;
+
 const DownVote = styled(ArrowDownwardIcon)`
   padding-left: 10px;
-  color: #f6da73;
-  transform: scale(1.4);
+  color: ${({ vote }) => (vote === "down" ? "#f6da73" : "#3e5336")};
+  transform: ${({ vote }) => (vote === "down" ? "scale(2.0)" : "scale(1.4)")};
 `;
 const RevWrap = styled.div`
   padding-left: 16px;
 `;
 
 function Review({
-  userName,
-  review_txt,
-  review_title,
-  upvotes,
-  downvotes,
+  logged_in_user,
+  user_id,
+  _id,
+  review,
+  title,
+  upvote_list,
+  downvote_list,
   date_posted,
-  score,
+  ratingVal,
 }) {
+  const check_voted = () => {
+    //console.log("check");
+    if (!logged_in_user) {
+      logged_in_user = "63713be424fbebf4c2789dca";
+    }
+    const up_index = upvote_list.indexOf(logged_in_user);
+    const down_index = downvote_list.indexOf(logged_in_user);
+    if (up_index > -1) {
+      setVote("up");
+      return true;
+    }
+    if (down_index > -1) {
+      console.log("down");
+      setVote("down");
+      return true;
+    }
+    return false;
+  };
+
+  const updateVoteLists = (isUpVote) => {
+    let new_downvotes;
+    let new_upvotes;
+    if (!logged_in_user) {
+      logged_in_user = "63713be424fbebf4c2789dca"; ////NOT A VALID USER ID
+    }
+    if (isUpVote) {
+      new_downvotes = downvote_list.filter((id) => id !== logged_in_user);
+      upvote_list.push(logged_in_user);
+      new_upvotes = upvote_list;
+    } else {
+      new_upvotes = upvote_list.filter((id) => id !== logged_in_user);
+      downvote_list.push(logged_in_user);
+      new_downvotes = downvote_list;
+    }
+    axios
+      .patch(`http://localhost:5000/reviews/${_id}`, {
+        upvote_list: new_upvotes,
+        downvote_list: new_downvotes,
+      })
+      .then((res) => console.log(res))
+      .catch((err) => console.log(err));
+  };
+
   const handleVote = (thisVote) => {
     if (!voted) {
       setVoted(true);
+      console.log("voted");
       setVote(thisVote);
       if (thisVote === "up") {
         setCurUpVote(curUpVote + 1);
       } else {
         setCurDownVote(curDownVote + 1);
       }
+      if (thisVote === "up") {
```

Comment:
You already have an if else above this with "up" or "down" so you can remove this one and add the lines to the above statements. Same for lines 149-153

Reply:
oh yep good catch, didn't even notice while I was shifting blocks around haha